tox.ini

```diff
@@ -10,7 +10,7 @@ sitepackages = False
 [base]
 deps =
     # recommended django version and other dependencies
-    Django>=1.8,<1.9
+    Django>=1.8,<2.0.2
```

Comment:
Too broad, please get latest minor release as before

Reply:
Please keep in mind that the current latest version of Django is  2.0.5

Reply:
Shouldn't we remove that line ? Lines under that one handle django versions.

Reply:
@jpic docs don't build without this line.